crates/oxc_linter/src/lib.rs

```diff
@@ -82,8 +83,19 @@ impl Linter {
         fix: bool,
     ) -> Vec<Message<'a>> {
         let ctx = LintContext::new(source_text, semantic.clone(), fix);
+        let trivias = semantic.trivias();
```

Comment:
We can create a `ESLintConfiguration::from(trivias)` to get all the info we need.